### PR TITLE
feat: copy bridge and firmware releases from connect-common

### PR DESCRIPTION
**Pull request by matejkriz**

### CODE REVIEW

**Comment by tsusanka** [comment]
This means we do not have to modify the releases.json here right? And we'll just change it in Suite in connect-common's `releases.json`.

**Comment by matejkriz** [comment]
> This means we do not have to modify the releases.json here right? And we'll just change it in Suite in connect-common's `releases.json`.

exactly, this deletes the releases.json files from here to have the single source of truth in Suite repo

File: package.json [hunk 1]
```diff
@@ -62,6 +62,7 @@
         "@babel/preset-env": "^7.12.10",
         "@babel/preset-flow": "^7.12.1",
         "@trezor/blockchain-link": "^1.0.17",
+        "@trezor/connect-common": "^0.0.2",
```
Comment (szymonlesisz) [inline package.json:65]: you need to add it to "extendedDependency" as well (to expose this dependency in npm-extended package)

File: src/js/env/node/networkUtils.js [hunk 2]
```diff
@@ -8,13 +8,34 @@ if (global && typeof global.fetch !== 'function') {
...
+    __dirname,
+    '../../../../node_modules/@trezor/connect-common/files/',
+);
+
+const getAbsolutePath = (fileUrl: string) => {
+    switch (fileUrl) {
+        case './data/bridge/releases.json':
+            return path.join(connectCommonFilesPath, 'bridge/releases.json');
+        case './data/bridge/latest.txt':
+            return path.join(connectCommonFilesPath, 'bridge/latest.txt');
```
Comment (szymonlesisz) [inline src/js/env/node/networkUtils.js]: im wondering if this file is even used (there is no such case in react-native)

**Review by szymonlesisz — CHANGES_REQUESTED** [review]

File: src/js/env/node/networkUtils.js (hunk 2 above)
Comment (matejkriz) [reply src/js/env/node/networkUtils.js]: 👍 deleted

File: package.json (hunk 1 above)
Comment (matejkriz) [reply package.json:65]: 👍 added

**Review by szymonlesisz — APPROVED** [review]

**Pull request merged**